cmd/create.go

```diff
@@ -48,16 +46,10 @@ var createCmd = &cobra.Command{
 			log.Println("Error running deployMetaphorCmd")
 			return err
 		}
-		// upload kubefirst config to user state S3 bucket
-		stateStoreBucket := viper.GetString("bucket.state-store.name")
-		err = aws.UploadFile(stateStoreBucket, config.KubefirstConfigFileName, config.KubefirstConfigFilePath)
-		if err != nil {
-			log.Printf("unable to upload Kubefirst cofiguration file to the S3 bucket, error is: %v", err)
-		}
-		log.Printf("Kubefirst configuration file was upload to AWS S3 at %q bucket name", stateStoreBucket)
+		aws.UploadKubefirstToStateStore(globalFlags.DryRun)
```

Comment:
I want to avoid too much logic on the create/init and ensure all are dry-tun safe.

Reply:
it's basically a one line call aws.UploadFile(), if we move it to a function, it will become:

```go
err := anotherFunc()
if err != nil {
...
}

func anotherFunc() error {
err := aws.UploadFile()
...
```

to make it even smaller we can do:
```
if err := aws.UploadFile(); if err != nil {
// log
}

Reply:
check the PR, I had to add it support dry-run mode. It has more line to keep logs in synch. 

![image](https://user-images.githubusercontent.com/53096417/187211326-db85ac95-12f1-45b1-a6f2-b953bcc68df0.png)


If you don't agree on it to be there, I will create another package for this function no worries.

Reply:
Are you removing the dryRun validation from the commands? if yes, then we can move it to `internal/state/state.go`

Reply:
remember to add the error , and return the error to the top caller